README.md

```diff
@@ -1 +1,50 @@
-# angular-ng-model-options-readme
\ No newline at end of file
+# ngModelOptions
+
+## Overview
+
+We've seen how to use `ng-model` to update our controller's value when a user types in a field, but we're used to controller updating immediately. `ngModelOptions` allows us to control *exactly* when these updates occur.
+
+## Objectives
+
+- Describe ngModelOptions
+- Use ngModelOptions on an input
+- Debounce Model updates
+- Control when $digest cycles occur
+
```

Comment:
Add a paragraph here explaining why we would want to control when the model is updating - what's the problem with the way we've been doing things so far?